app/helpers/order_cycles_helper.rb

```diff
@@ -38,13 +38,13 @@ def permitted_hub_enterprise_options_for(order_cycle)
 
   def order_cycle_status_class(order_cycle)
     if order_cycle.undated?
-      'undated'
+      I18n.t('admin.order_cycles.status.undated')
```

Comment:
The strings from this method are used as a CSS class to style table rows:
https://github.com/openfoodfoundation/openfoodnetwork/blob/fbf4513e96e52fe89ca4e7de421fabfc2e8e15c9/app/views/admin/order_cycles/_row.html.haml#L1

Translating these strings would break this. You probably want to do the translation here instead:
https://github.com/openfoodfoundation/openfoodnetwork/blob/fbf4513e96e52fe89ca4e7de421fabfc2e8e15c9/app/assets/javascripts/admin/orders/controllers/order_controller.js.coffee#L23

We have a Javascript method called `I18n.t` for this.